### ISSUE TITLE
[BUG]: EACCESS : permission denied on file upload

### ISSUE THREAD

**Issue by boooon95** [Mintplex-Labs/anything-llm#1913]
### How are you running AnythingLLM?

Docker (remote machine)

### What happened?

Hello everyone,

I am trying to install anything-llm in a self-hosted setup on Alma Linux. Everything is going well and it works fine without RAG. However, when I try to add a file, I get the following error:
"Invalid file upload. EACCES: permission denied, open '/app/collector/hotdir/test.txt'"

I tried to change the permissions on the "documents" directory, but nothing works. I also tried to follow the guidance provided in [#821](https://github.com/Mintplex-Labs/anything-llm/issues/821), but I cannot find the correct directory structure.

I am using J'utilse ollama as LLM provider and lanceDB for the database

Any idea ?

**Comment by warrenwingaru** [comment]
Experiencing this as well, Have you figured it out?

**Comment by boooon95** [comment]
Still no luck, but I am continuing to search

**Comment by warrenwingaru** [comment]
Found a solution that works well, you could volume mount the `/app/collector/hotdir` directory to host path. Files stored there are temporary anyways.

**Comment by boooon95** [comment]
Thank you so much, it works great !
For those who are interested, here is a modified version of the command to run anything-llm on Linux using Docker:
```
export STORAGE_LOCATION=$HOME/anythingllm && \
mkdir -p $STORAGE_LOCATION && \
mkdir -p $STORAGE_LOCATION/hotdir && \
touch "$STORAGE_LOCATION/.env" && \
docker run -d -p 3001:3001 \
--name llm \
--cap-add SYS_ADMIN \
-v ${STORAGE_LOCATION}:/app/server/storage \
-v ${STORAGE_LOCATION}/hotdir:/app/collector/hotdir \
-v ${STORAGE_LOCATION}/.env:/app/server/.env \
-e STORAGE_DIR="/app/server/storage" \
mintplexlabs/anythingllm
```

**Comment by ragonneau** [comment]
I am curious as to why this PR is closed... This looks to me like an actual bug of AnythingLLM.

**Comment by timothycarambat** [comment]
It has already been fixed by #1915
